README.md

```diff
@@ -1,20 +1,59 @@
-# jQuery-FileUpload
+![](https://heatbadger.now.sh/github/readme/contributte/fileupload/)
 
-[![Latest stable](https://img.shields.io/packagist/v/jzechy/jquery-fileupload.svg?style=flat-square)](https://packagist.org/packages/jzechy/jquery-fileupload)
-[![license](https://img.shields.io/github/license/jzechy/jquery-fileupload.svg?maxAge=2592000&style=flat-square)](https://github.com/JZechy/jQuery-FileUpload/blob/master/LICENSE)
-[![Downloads Total](https://img.shields.io/packagist/dt/jzechy/jquery-fileupload.svg?style=flat-square)](https://packagist.org/packages/jzechy/jquery-fileupload)
-[![Open Issues](https://img.shields.io/github/issues/jzechy/jquery-fileupload.svg?style=flat-square)](https://github.com/JZechy/jquery-fileupload/issues)
+<p align=center>
+  <a href="https://github.com/contributte/fileupload/actions"><img src="https://badgen.net/github/checks/contributte/fileupload/master?cache=300"></a>
+  <a href="https://coveralls.io/r/contributte/fileupload"><img src="https://badgen.net/coveralls/c/github/contributte/fileupload?cache=300"></a>
+  <a href="https://packagist.org/packages/jzechy/jquery-fileupload"><img src="https://badgen.net/packagist/dm/jzechy/jquery-fileupload"></a>
+  <a href="https://packagist.org/packages/jzechy/jquery-fileupload"><img src="https://badgen.net/packagist/v/jzechy/jquery-fileupload"></a>
+</p>
+<p align=center>
+  <a href="https://packagist.org/packages/jzechy/jquery-fileupload"><img src="https://badgen.net/packagist/php/jzechy/jquery-fileupload"></a>
+  <a href="https://github.com/contributte/fileupload"><img src="https://badgen.net/github/license/contributte/fileupload"></a>
+  <a href="https://bit.ly/ctteg"><img src="https://badgen.net/badge/support/gitter/cyan"></a>
+  <a href="https://bit.ly/cttfo"><img src="https://badgen.net/badge/support/forum/yellow"></a>
+  <a href="https://contributte.org/partners.html"><img src="https://badgen.net/badge/sponsor/donations/F96854"></a>
+</p>
 
-jQuery-FileUpload je komponenta, fungující jako rozšíření UploadControl v Nette formuláři, která pro nahrávání souborů využívá [blueimp/jQuery-File-Upload](https://github.com/blueimp/jQuery-File-Upload).
+<p align=center>
+Website 🚀 <a href="https://contributte.org">contributte.org</a> | Contact 👨🏻‍💻 <a href="https://f3l1x.io">f3l1x.io</a> | Twitter 🐦 <a href="https://twitter.com/contributte">@contributte</a>
+</p>
 
-Komponenta nabízí možnost [vlastního uživatelského rozhraní](https://github.com/JZechy/jQuery-FileUpload/wiki/Rendering) a [vlastního zpracování nahrávaných souborů](https://github.com/JZechy/jQuery-FileUpload/wiki/Upload-model).
+## Usage
+
+To install latest version of `jzechy/flysystem` use [Composer](https://getcomposer.org/).
```

Comment:
Tady to ma byt spis jzechy/jquery-fileupload

Reply:
opraveno